client/app/scripts/superdesk/services/modalService.js

```diff
@@ -43,5 +43,45 @@ define(['angular'], function(angular) {
                 return delay.promise;
             };
 
+        }])
+        .directive('sdModal', ['$document', function($document) {
+
+            return {
+                replace: true,
+                template: [
+                    '<div class="modal fade hide">',
+                    '<div class="modal-dialog" ng-if="isModalOpen === \'true\'"><div class="modal-content" ng-transclude></div></div>',
+                    '</div>'].join(''),
+                transclude: true,
+                link: function(scope, element, attrs) {
+
+                    var par = element.parent();
+
+                    attrs.$observe('open', function(model) {
+                        scope.isModalOpen = attrs.open;
+                        if (model === 'true') {
+                            element.appendTo($document.find('body'));
```

Comment:
it's called every time you run a modal, but it's also paired with close modal function, when you return content to original place